bindings/python/meson.build

```diff
@@ -0,0 +1,26 @@
+project(
+  'tree-sitter-capnp',
+  'c',
+  license: 'MIT'
+)
```

Comment:
I would add also:

```meson
meson_version: '>=0.64.0',
```
since you are using functionality that depends on this. While pip install will typically always install the latest version, this lets meson produce a better error message e.g. on LTS distros with an older meson available.

If you set this to `meson_version: '>=0.37.0'` (chosen for no particular reason other than that's when we started tracking features) you get this warning:

```
meson.build:8: WARNING: Project targets '>=0.37.0' but uses feature introduced in '0.46.0': module python.
meson.build:8: WARNING: Project targets '>=0.37.0' but uses feature introduced in '0.51.0': modules arg in python.find_installation.
meson.build:8: WARNING: Project targets '>=0.37.0' but uses feature introduced in '0.64.0': pure arg in python.find_installation.

[...]

meson.build:19: WARNING: Project targets '>=0.37.0' but uses feature introduced in '0.49.0': / with string arguments.
meson.build:19: WARNING: Project targets '>=0.37.0' but uses feature introduced in '0.49.0': / with string arguments.
meson.build:19: WARNING: Project targets '>=0.37.0' but uses feature introduced in '0.49.0': / with string arguments.
meson.build:16: WARNING: Project targets '>=0.37.0' but uses feature introduced in '0.48.0': gnu_symbol_visibility arg in build target.
meson.build:27: WARNING: Project targets '>=0.37.0' but uses feature introduced in '0.49.0': / with string arguments.
```

and a final summary:

```
WARNING: Project specifies a minimum meson_version '>=0.37.0' but uses features which were added in newer versions:
 * 0.46.0: {'module python'}
 * 0.48.0: {'gnu_symbol_visibility arg in build target'}
 * 0.49.0: {'/ with string arguments'}
 * 0.51.0: {'modules arg in python.find_installation'}
 * 0.64.0: {'pure arg in python.find_installation'}
```
This makes it easy to quickly check which version of the meson.build API you need.